pkg/k8shandler/configuration_tmpl.go

```diff
@@ -19,11 +19,11 @@ network:
 
 discovery.zen:
   ping.unicast.hosts: {{.EsUnicastHost}}
-  minimum_master_nodes: ${NODE_QUORUM}
+  minimum_master_nodes: {{.NodeQuorum}}
```

Comment:
how was `${NODE_QUORUM}` evaluated previously?  As an environment variable at runtime?  Which means the user could change the value by changing the deployment env var?

Reply:
Yes, it was an environment variable. We're setting the quorum dynamically now based on the number of master nodes in the cluster: `(master_eligible_nodes / 2) + 1`

https://www.elastic.co/guide/en/elasticsearch/reference/6.4/discovery-settings.html#minimum_master_nodes